extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -10,6 +10,15 @@ class CommunityPageSpecialController extends WikiaSpecialPageController {
 	const MODAL_IMAGE_HEIGHT = 700.0;
 	const MODAL_IMAGE_MIN_RATIO = 0.85;
 	const DEFAULT_MODULES_MAX = 3;
+	// order of permissions is consistent with badges hierarchy in Discussions
+	const PERMISSIONS_HIERARCHY = [
```

Comment:
I wanted to stress here that the order here is not random, it has its purpose and shouldn't be changed. What about `BADGE_HIERARCHY` or `BADGE_GROUP_HIERARCHY`?